src/main/java/com/womply/billing/killbill/plugins/db/AuthorizeNetDAOImpl.java

```diff
@@ -214,6 +214,12 @@ protected SelectConditionStep getPaymentMethodForKauiQuery(UUID kbAccountId, UUI
                         .as(AuthorizeNetPaymentMethod.KAUI_FIELD_CARD_EXPIRATION_YEAR),
                 AUTHORIZE_NET_PAYMENT_METHODS.CC_LAST_4
                         .as(AuthorizeNetPaymentMethod.KAUI_FIELD_CARD_LAST_4),
+                AUTHORIZE_NET_PAYMENT_METHODS.ROUTING_NUMBER
+                        .as(AuthorizeNetPaymentMethod.KAUI_FIELD_ACH_ROUTING_NUMBER),
+                AUTHORIZE_NET_PAYMENT_METHODS.INSTITUTION_NAME.as(
+                        AuthorizeNetPaymentMethod.KAUI_FIELD_ACH_INSTITUTION_NAME),
+                AUTHORIZE_NET_PAYMENT_METHODS.CC_LAST_4
+                        .as(AuthorizeNetPaymentMethod.KAUI_FIELD_ACH_ACCOUNT_NUMBER_LAST_4),
```

Comment:
why is CC_LAST_4 == ACH_ACCOUNT_NUMBER_LAST_4?

Reply:
We thought `CC_LAST_4` could be leveraged to store the last 4 of the ACH (to avoid extra refactoring effort).

Reply:
You can see here that the KillBill guys are doing the samething (i.e. leveraging a CC-centric table to store ACH info): https://github.com/killbill/killbill-stripe-plugin/blob/master/db/schema.rb#L10-L24

Reply:
personal opinion, but I would not conflate the 2. now cc-last-4 doesn't mean credit card anymore, and I have to go look at another field to determine what account type these digits apply to

Reply:
Would you feel a bit at ease if we removed the `cc_` prefix for the shared fields?

Reply:
+1 what tyler said; we should not rename this field. I lean toward adding a new field for ACH like phil suggested, but at the very least we should not change the name of the field that already exists.

Reply:
So the way I see it is we are trying to put lipstick on a pig's lips. The setup that we have is already  non-ideal. If we are talking about adding `ach_last_4` (which adds work for nothing IMO), then how about adding `ach_first_name`, `ach_last_name`, etc (basically an `ach_`-prefixed counterpart for every `cc_`-prefixed field; next, let's start talking about having a different table... and the scope would just keep increasing.

I would propose we leave this as is, and after we deliver our ACH MVP, we can sit down and have a fruitful discussion about not only how to resolve this, but also how to prevent this situation from reoccurring, and how to introduce changes given that the community is using our repo.

Reply:
If we're going to re-use the db column, let's also re-use the member variable.  the `select cc_last_4 as ach_last_4` is making this even weirder than it already is.

and then we can pretend that `cc` stands for a`cc`ount !

Reply:
can we just rename this to account_last_4 and use for both if the rows already have the distinction between the types? I agree that we should not overload the name. I just spent 2 hours in the womply hash code base trying to implement something because the variable name made it sound like there was a binary encoding where in fact it was a hex encoding. Is it hard to just name this account_last_4?

Reply:
Addressed in the latest force pushed commit.